src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.figures;
+
+public class IsoscelesTrapezoid extends Figure {
+
+    private final double firstBase;
+    private final double secondBase;
+    private final double height;
+
+    public IsoscelesTrapezoid(String color, double firstBase, double secondBase, double height) {
+        super(color);
+        this.firstBase = firstBase;
+        this.secondBase = secondBase;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.round(((firstBase + secondBase) * height / 2) * 10.0 / 10.0);
```

Comment:
The use of `Math.round` here is unnecessary if you are not looking to round the value to a certain number of decimal places. If you wish to round the area to a certain number of decimal places, you should specify the precision. For example, to round to two decimal places, you can use `Math.round(value * 100.0) / 100.0`. Make sure to remove any magic numbers and replace them with named constants as per the checklist.